### PR TITLE
Covid-19 command

### PR DESCRIPTION
**Pull request by Doges**
Adds the covid19 command including states, continents, worldwide and info from countries. Also special thanks to https://github.com/disease-sh/ for providing the API.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "abort-controller": "^2.0.3",
     "ascii-table": "0.0.9",
     "axios": "^0.20.0",
+    "brainfuck-node": "^1.0.2",
```
Comment (almeidx) [inline package.json:41]: 🤔

Comment (Doges) [reply package.json:41]: https://github.com/SwitchbladeBot/switchblade/pull/1226/files

they not added the module itself, that's why i added it

File: src/commands/misc/covid.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+module.exports = class Covid extends SubcommandListCommand {
+  constructor (client) {
+    super({
+      name: 'covid',
+      aliases: ['covid19', 'coronavirus'],
+      authorString: 'commands:covid.covid',
+      authorImage: 'https://i.imgur.com/Rnobe3k.png',
+      authorURL: 'https://covid19.who.int/',
+      embedColor: Constants.NPM_COLOR
```
Comment (almeidx) [inline src/commands/misc/covid.js:11]: Add a separate color for this command instead of using NPM's

File: src/commands/misc/covid/continents.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const { Command, CommandError, SwitchbladeEmbed, MiscUtils, Constants } = require('../../../')
+
```
Comment (almeidx) [inline src/commands/misc/covid/continents.js:2]: This new line should be removed to be consistent with other files

File: src/commands/misc/covid/continents.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        missingError: 'commands:covid.subcommands.continents.noContinent'
+      }]
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }, continent) {
+    moment.locale(language)
+    try {
+      const { data } = await this.client.apis.covid.getContinent(continent)
+      console.log(data)
```
Comment (almeidx) [inline src/commands/misc/covid/continents.js:23]: ❌

File: src/commands/misc/covid/continents.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      channel.send(
+        new SwitchbladeEmbed(author)
+          .setColor(Constants.NPM_COLOR)
```
Comment (almeidx) [inline src/commands/misc/covid/continents.js:26]: Same as the other comment

File: src/commands/misc/covid/countries.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
+const { Command, CommandError, SwitchbladeEmbed, MiscUtils, Constants } = require('../../../')
+
```
Comment (almeidx) [inline src/commands/misc/covid/countries.js:2]: This new line should be removed as well

File: src/commands/misc/covid/countries.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  async run ({ t, author, channel, language }, country) {
+    moment.locale(language)
+    try {
+      const { data } = await this.client.apis.covid.getCountry(country)
+      channel.send(
+        new SwitchbladeEmbed(author)
+          .setThumbnail(data.countryInfo.flag)
+          .setColor(Constants.NPM_COLOR)
```
Comment (almeidx) [inline src/commands/misc/covid/countries.js:26]: Same as the other comments

File: src/commands/misc/covid/states.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
+const { Command, CommandError, SwitchbladeEmbed, MiscUtils, Constants } = require('../../../')
+
```
Comment (almeidx) [inline src/commands/misc/covid/states.js:2]: This new line should be removed

File: src/commands/misc/covid/states.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }, state) {
+    moment.locale(language)
+    try {
+      const { data } = await this.client.apis.covid.getState(state)
+      channel.send(
+        new SwitchbladeEmbed(author)
+          .setColor(Constants.NPM_COLOR)
```
Comment (almeidx) [inline src/commands/misc/covid/states.js:25]: Same as the other comments

File: src/commands/misc/covid/worldwide.js [hunk 10]
```diff
@@ -0,0 +1,37 @@
+const { Command, CommandError, SwitchbladeEmbed, MiscUtils, Constants } = require('../../../')
+
```
Comment (almeidx) [inline src/commands/misc/covid/worldwide.js:2]: This new line should be removed

File: src/commands/misc/covid/worldwide.js [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }) {
+    moment.locale(language)
+    try {
+      const { data } = await this.client.apis.covid.getWorldwide()
+      channel.send(
+        new SwitchbladeEmbed(author)
+          .setColor(Constants.NPM_COLOR)
```
Comment (almeidx) [inline src/commands/misc/covid/worldwide.js:20]: Same as the other comments

**Review by almeidx — DISMISSED** [review]

File: src/commands/misc/covid.js (hunk 2 above)
Comment (Doges) [reply src/commands/misc/covid.js:11]: theres no problem using the npm actually, i wanted to use a red color.

**Review by almeidx — APPROVED** [review]

File: index.js [hunk 12]
```diff
@@ -25,6 +25,5 @@ console.log(readFileSync('bigtitle.txt', 'utf8').toString())
 
 const Switchblade = require('./src/Switchblade.js')
 const client = new Switchblade(CLIENT_OPTIONS)
-client.on('debug', (...args) => console.log('debug', ...args))
```
Comment (fernando457829) [inline index.js:28]: Opss

**Review by fernando457829 — CHANGES_REQUESTED** [review]

File: index.js (hunk 12 above)
Comment (fernando457829) [reply index.js:28]: Not-Opss

**Review by fernando457829 — APPROVED** [review]

**Pull request merged**